File: 58-LengthOfLastWord/58-LengthOfLastWord.java
// Last updated: 20/07/2026, 18:40:10
class Solution {
    public int lengthOfLastWord(String s)
     {
        String str=s.trim();
        int count=0;
        for(int i=str.length()-1;i>=0;i--)
        {
            if(str.charAt(i)!=' ')
            {
                count=count+1;
            }
            else
            {
                break;
            }
        }
        return count;
        
    }

public static void main(String args[])
{
    Solution solution=new Solution();
    String str="Hello World";
    System.out.println(solution.lengthOfLastWord(str));
}
}

File: 58-LengthOfLastWord/58-LengthOfLastWord.py
# Last updated: 20/07/2026, 18:40:05
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count=0
        for i in range(len(s)-1,-1,-1):
            if count!=0 and s[i]==" ":
                break
            elif s[i]==" ":
                continue
            else:
                count+=1
            
        return count
        


        